### DISCUSSION TITLE
Is there any intention for the Badge component to support digital display?

### DISCUSSION THREAD

**Discussion by AmelloAster** [mantinedev/mantine#2231] [Feature requests]
Just like this
<img width="299" alt="image" src="https://user-images.githubusercontent.com/26178326/186608594-0fb0080d-8718-4e03-9f8f-614ab920a778.png">
This UI scenario should also be considered common, but from the current design of Badge, it seems that this scenario is not considered?Will there be a follow-up to this plan?

**Comment by rtivital** [comment]
https://mantine.dev/core/indicator/

**Reply by AmelloAster** [reply]
> https://mantine.dev/core/indicator/

OK，Thank you
